python/tvm/contrib/target/__init__.py

```diff
@@ -14,5 +14,3 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Codegen and runtime APIs for targets.
```

Comment:
Just confused why the annotation has been removed : )

Reply:
Removed this as per the review comment from @zhiics https://github.com/apache/incubator-tvm/pull/5052#discussion_r439504954